.github/workflows/php.yml

```diff
@@ -1,5 +1,5 @@
 name: PHP tests
-on: [push, pull_request]
+on: [push]
```

Comment:
Why not running these actions on pull requests?

Reply:
I prefer on push to directly have the issues reported if there is some